RA_Integration.sln

```diff
@@ -14,17 +14,27 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "RA_Integration.Tests", "tes
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Analysis|Win32 = Analysis|Win32
+		AnalysisUnicode|Win32 = AnalysisUnicode|Win32
```

Comment:
Put in new Analysis Configuration, Test projects is Unicode only. They are both debug, doesn't make sense to use it for release mode.